package.json

```diff
@@ -50,6 +50,7 @@
     "minimatch": "^2.0.4"
   },
   "ember-addon": {
-    "configPath": "tests/dummy/config"
+    "configPath": "tests/dummy/config",
+    "before": "ember-cli-deploy-redis"
```

Comment:
@lukemelia This does feel a little weird in that it fixes it for this example but if a user is using a different plugin uploading the index.html somewhere else, they have the same problem.  The "advanced config" will really be the true solution to this I think where a user can specify in config the order that plugins should run.  But for now, this is a good middle ground.